vscode/microsoft-kiota/src/steps.ts

```diff
@@ -312,7 +309,8 @@ export async function generateSteps(existingConfiguration: Partial<GenerateState
         }
     }
     async function inputPluginName(input: MultiStepInput, state: Partial<GenerateState>) {
-        if (!state.pluginName){
+        const isDeepLinkPluginNameProvided = deepLinkParams?.source && deepLinkParams.source.toLowerCase() === 'ttk' && deepLinkParams.name && state.pluginName;
```

Comment:
Deeplinking is not limited to ttk. It can be API center , tafuta or any other source. We could correct this check by checking if deeplinking is enabled, which in my opinion would translate to  if (Object.Keys(deepLinkParams)>0). Then if deeplinking is enabled and state.pluginName is enabled then skip asking plugin name. Otherwise show the prompt.